tests/ChunkingRedisClient.Tests/ChunkingRedisClient.Tests.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Can this be changed to "IntegrationTests" - Eventually I'll be introducing some unit tests and I prefer to keep these in separate assemblies (so that automated test runs can target the "runnable" tests for the environment)

Reply:
Yep! makes sense!